neonvm-runner/cmd/main.go

```diff
@@ -225,13 +225,33 @@ func run(logger *zap.Logger) error {
 
 	// create iso9660 disk with runtime options (command, args, envs, mounts)
 	tg.Go("iso9660-runtime", func(logger *zap.Logger) error {
+		disks := vmSpec.Disks
+
+		// add the tls path.
+		// this is needed to just `mkdir` the mounting directory.
+		if vmSpec.TLS != nil {
+			watch := true
+			disks = append(disks, vmv1.Disk{
+				Name:      "tls-keys",
+				MountPath: vmSpec.TLS.MountPath,
+				Watch:     &watch,
```

Comment:
```suggestion
				Watch:     lo.ToPtr(true),
```

(we tend to use `github.com/samber/lo` to help make this sort of thing cleaner)